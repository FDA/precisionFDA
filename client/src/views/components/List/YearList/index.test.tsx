import React from 'react'
import { shallow } from 'enzyme'

import { YearList } from '.'


describe('YearList', () => {
    it('matches snapshot', () => {
      const wrapper = shallow(<YearList setYearHandler={() => {}} />)
  
      expect(wrapper).toMatchSnapshot()
    })
})