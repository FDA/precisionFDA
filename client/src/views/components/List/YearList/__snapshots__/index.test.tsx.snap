// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YearList matches snapshot 1`] = `
<QueryList
  className="year-list"
  emptyMessage="No previous years"
  listExtractor={[Function]}
  query={[Function]}
  template={[Function]}
/>
`;
